.github/workflows/build-test-deploy-dev.yml

```diff
@@ -0,0 +1,272 @@
+name: Build and deploy to IPFS
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    branches: ['main']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: |
+      (github.event_name == 'push' && github.event_name == 'pull_request') ||
+      (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name &&
+      !contains(github.head_ref, 'dependabot'))
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v3.0.2
+        with:
+          persist-credentials: false
+
+      - name: Build App
+        uses: ./.github/actions/build
+        with:
+          NEXT_PUBLIC_ENV: 'prod'
+          NEXT_PUBLIC_FORK_BASE_CHAIN_ID: ${{ secrets[format('FORK_BASE_CHAIN_ID_{0}', github.event.pull_request.number)] }}
+          NEXT_PUBLIC_FORK_CHAIN_ID: ${{ secrets[format('FORK_CHAIN_ID_{0}', github.event.pull_request.number)]  }}
+          NEXT_PUBLIC_FORK_URL_RPC: ${{ secrets[format('FORK_URL_RPC_{0}', github.event.pull_request.number)]  }}
+          NEXT_PUBLIC_MIXPANEL: ${{ secrets.NEXT_PUBLIC_MIXPANEL }}
+          NEXT_PUBLIC_FIAT_ON_RAMP: 'false'
+          NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID: ${{ secrets.WALLET_CONNECT_PROJECT_ID }}
+
+      - name: Upload artifacts
+        uses: ./.github/actions/upload-artifacts
+        with:
+          BUILD_ARTIFACT_NAME: 'out'
+
+  build_staging:
```

Comment:
I think for this workflow we wouldn't need the `build_staging` job.

Reply:
yeah agreed, we don't use staging really so we can remove this